integration/e2e/e2e_test.go

```diff
@@ -489,6 +489,8 @@ var _ = Describe("EndToEnd", func() {
 			ordererRunner := network.OrdererRunner(orderer)
 			ordererProcess = ifrit.Invoke(ordererRunner)
 			Eventually(ordererProcess.Ready, network.EventuallyTimeout).Should(BeClosed())
+			Eventually(ordererRunner.Err(), network.EventuallyTimeout).Should(gbytes.Say("Setting up cluster when there is no system channel: assuming cluster-type consensus"))
```

Comment:
In the context of each file, these logs seemed to make sense, but it seems a bit redundant to see what's essentially the same information logged three times.

Reply:
Alas, that is the way I found to ensure the changes I made to the consenter and main.go are in effect... any suggestions?

Reply:
Perhaps we switch all but one of these messages to debug level and just set the necessary loggers to debug for this test? And maybe include a TODO (or create a JIRA cleanup task) to remove the messages once some of the other functionality is place for us to actually test that the orderer is usable without a system channel.

Reply:
I have a solution: I simply verify that the registrar has an etcdraft consenter when it is initialized without a system channel, and throw panic if it does not.